xcode/Subconscious/Shared/Data/Geists/zettelkasten.json

```diff
@@ -21,6 +21,13 @@
       "What questions does this answer?",
       "What new questions does this trigger?"
   ],
+  "combo_followup": [
```

Comment:
@bfollington let's create a separate `combo.json` grammar for the combo geist.

Reply:
Done!